### ISSUE TITLE
Code folding and guides disappearing when accessibility is turned on

### ISSUE THREAD

**Issue by getsetgopi** [Microsoft/vscode#70250]
- VSCode Version: 1.32.1
- OS Version: Windows 10

Steps to Reproduce:
1. Go to settings
2. Search for Accessibility support
3. If turned off. Turn it on
4. Open any html/css/js/jsx/ts etc. file.
5. Observer the code folding icon (+/-) and tab guides are no longer visible

Expected result: Even with the accessibility support on. The code folding and guides should be visible.

Does this issue occur when all extensions are disabled?: Yes

**Comment by yume-chan** [comment]
This working as intended.

> Certain features such as folding, minimap (code overview), and word wrap are disabled when in screen reader mode.

(from https://code.visualstudio.com/docs/editor/accessibility#_screen-reader-mode)

https://github.com/Microsoft/vscode/issues/32166#issuecomment-331932055 contains the reason.

**Comment by getsetgopi** [comment]
Thanks for the confirmation.
